README.md

```diff
@@ -170,105 +201,81 @@ These instructions have been written with a Debian 12 machine in mind, but they
-```
-#### Dedicated Server on Windows
-These instructions were tested on Windows Server 2016 and Windows 11 machines, they will probably work in all Windows versions.
-1. Install SteamCMD following these instructions: [LINK](https://developer.valvesoftware.com/wiki/SteamCMD#Windows)
-2. Choose a location for your server to be installed into, for example, `C:\NeotokyoServer\`, and create it. (In this case SteamCMD is also installed in this location)
-3. Run SteamCMD: `steamcmd.exe`
-4. Enter the following commands in SteamCMD:
-    ```
-    force_install_dir .\ognt\
-    login anonymous
-    app_update 313600 validate
-    (wait for it to install)
-    quit
-    ```
-5. Run SteamCMD again, and enter these commands:
-    ```
-    force_install_dir .\ntrebuild\
-    (this will be the main directory of your server)
-    login anonymous
-    app_update 244310 validate
-    (wait for it to install)
-    quit
-    ```
-6. Extract the latest release of NT;RE into `<YOUR_LOCATION>\ntrebuild`, so you will have a directory `<YOUR_LOCATION>\ntrebuild\neo` with a `gameinfo.txt` inside.
-7. Allow all Inbound and Outbound TCP and UDP requests for the following ports via Windows Firewall. [See how](https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/configure#create-an-inbound-port-rule)
-    * 27015 TCP+UDP (you can keep the TCP port closed if you don't need RCON support)
-    * 27020 UDP
-    * 27005 UDP
-    * 26900 UDP
-8. Your server should be ready to go, launch it inside your main directory (`(...)\ntrebuild\`) with the following command: (You can alter any argument to your liking, except `-game` and `-neopath`) 
-```
-srcds.exe -game neo -neopath "..\ognt\NeotokyoSource" +ip <YOUR_IP> -maxplayers <1-32> +map <MAP_NAME>
-```
+4. `cd` up directories twice, so that you will be in `<YOUR_LOCATION>/ntrebuild`.
+
+#### Extracting NT;RE
+Extract the latest release of NT;RE into `<YOUR_LOCATION>/ntrebuild`, so you will have a directory `<YOUR_LOCATION>/ntrebuild/neo` with a `gameinfo.txt` inside.
+
+##### Developer-only: Mounting NT;RE
+If you're developing and just testing the server, it may be more useful to mount it to your `game/neo` from the git checkout.
+
+* Windows: `mklink /J neo "<path_to_source>/game/neo"`
+* Linux: Non-persistent bind mount: `mkdir neo && sudo mount --bind <path_to_source>/game/neo neo`
 
-### Testers/Devs
-1. To run a server, install "Source SDK Base 2013 Dedicated Server" (appid 244310).
-2. For firewall, open the following ports:
+Or just use the directory directly when passing to the srcds command line arguments: `-game <path_to_source>/game/neo`.
+
+#### Exposing the server to the wider internet
+
+There are two ways on exposing your dedicated server to the wider internet: Port forwarding and Steam Networking.
+
+##### Firewall - Port forwarding
+* Open the following ports:
```

Comment:
NIT: We should discourage this a little more in the guide; if we are given a free way to prevent randoms from having access to your server ip address, I feel it should be encouraged and people should understand the risks of continuing to use this depreciated method.